.github/workflows/catalogs.yml

```diff
@@ -63,6 +63,31 @@ jobs:
           path: azure.csv
           retention-days: 1
 
+  catalog-nebius:
```

Comment:
Leftovers from copypaste, I guess catalog-datacrunch was supposed

Reply:
Yes this was copypaste. I fixed it.